tests/core/object/test_class_db.h

```diff
@@ -623,6 +623,11 @@ void add_exposed_classes(Context &r_context) {
 				arg.name = orig_arg_name;
 				arg.position = i;
 
+#ifdef DEBUG_METHODS_ENABLED
+				TEST_FAIL_COND(!arg.name.is_valid_identifier(),
+						"Invalid method argument name: '", arg.name, "'  '", exposed_class.name, ".", method.name, "'.");
+#endif //  DEBUG_METHODS_ENABLED
```

Comment:
```suggestion
#endif // DEBUG_METHODS_ENABLED
```
Will fix once CI is done